src/core/utils/pickers.ts

```diff
@@ -4,38 +4,62 @@ import type { DefaultKeys, DefaultStylesConfigs } from '../../defaultConfig/type
 import type { KeyType } from '../../types/misc'
 import { Constants } from '../config'
 
-type GetOffsetTopProps = {
+type GetNotificationOffsetProps = {
   globalConfig: NotificationsConfig<VariantsMap>
   notificationHeight: number
   isPortraitMode: boolean
   windowHeight: number
   statusBarHeight: number
 }
 
-export const getTopOffset = ({
+export const getNotificationOffset = ({
   globalConfig,
   notificationHeight,
   isPortraitMode,
   windowHeight,
   statusBarHeight,
-}: GetOffsetTopProps) => {
+}: GetNotificationOffsetProps) => {
   const isNotch = globalConfig.isNotch
   const extraSpace = statusBarHeight + 10
 
   const shouldRenderExtraSpace = isNotch ?? (isPortraitMode && !Constants.isAndroid)
   const topPosition = shouldRenderExtraSpace ? extraSpace : 10
   const notificationPosition = globalConfig.notificationPosition
 
+  let topOffset, leftOffset, rightOffset
+
   switch (notificationPosition) {
     case 'top':
-      return topPosition
+      topOffset = topPosition
+      break
+    case 'top-left':
+      topOffset = topPosition
+      leftOffset = 10
+      break
+    case 'top-right':
+      topOffset = topPosition
+      rightOffset = 10
```

Comment:
I was wondering if we could use here Constants.notificationSideMargin, that will be more consistent. 
What I mean by that:
1. Full screen notification has side margins with 14px each, value from the Constants
2. Side screen notification will be with margin 10 from left, or right

In that case these 'margins' should be treated as left/right padding of the entire screen, and should be te same for each element.

What are your thoughts on it? 🤔

Reply:
Good point. I've adjusted the margins to take values from the Constants and all margins now are the same.